app/models/game.rb

```diff
@@ -13,28 +13,28 @@ class Game < ApplicationRecord
 # Game rules of chess
 def initialize_board_white_pieces
   8.times do
-    Pawn.create(game_id: id, x_position: 2, y_position: 1, color: true)
+    Pawn.create(game_id: id, x_position: 1, y_position: 2, color: 'white')
```

Comment:
What happened to the `index` you had? This will only create 8 pawns at coordinates (1,2), we should have one each at (2,2), (3,2), (4,2) etc.